src/static/js/modules/explore-rates.js

```diff
@@ -1195,6 +1207,11 @@ $( '.calc-loan-amt, .credit-score' ).on( 'keyup', '.recalc', function( evt ) {
   }
 } );
 
+$( '.calc-loan-amt, .credit-score' ).on( 'blur', '.recalc', function( evt ) {
```

Comment:
Good call, I made this way more specific and changed the on blur to focus out since the changing on blur affects how people arrow back to change the values in e4a8d51